tests/scorers/test_hallucination_scorer.py

```diff
@@ -1,49 +1,63 @@
+import json
+
 import pytest
-from openai import OpenAI
+from pydantic import BaseModel
 
 import weave
 from weave.scorers import (
     HallucinationFreeScorer,
 )
 from weave.scorers.hallucination_scorer import (
-    HallucinationReasoning,
     HallucinationResponse,
 )
 
 
-# mock the create function
+# Mock the acompletion function
 @pytest.fixture
-def mock_create(monkeypatch):
-    def _mock_create(*args, **kwargs):
-        return HallucinationResponse(
-            chain_of_thought="The output is consistent with the input data.",
-            reasonings=[
-                HallucinationReasoning(
-                    observation="My observation for this is that the output is consistent with the input data.",
-                    hallucination_type="No Hallucination",
-                )
+def mock_acompletion(monkeypatch):
```

Comment:
Is this acompletion only used inside the hallucination scorer?  If yes, move it inside that fixture